README.md

```diff
@@ -3,6 +3,9 @@
 This repository is about presenting `libvlc` and its capabilities to .NET developers.
 It also contains packaging tools and files for nuget packaging/deployment.
 
+In other words : It's just the same thing as you'd download from VideoLAN's website, in a NuGet package,
```

Comment:
Suggestion: It's just the same thing as if you had downloaded the files from VideoLAN's website, in a NuGet package,